### PR TITLE
fix quarterly console error#177

### PR DESCRIPTION
**Pull request by sgrg1**
fix issue with elements with non-unique id for currency input

### CODE REVIEW

File: src/components/PaymentField.jsx [hunk 1]
```diff
@@ -13,7 +13,7 @@ const CustomInputComponent = ({
   ...props
 }) => {
   const { name } = field;
-  const { className, isNegativeValue, label } = props;
+  const { className, isNegativeValue, label,month } = props;
```
Comment (martypowell) [inline src/components/PaymentField.jsx]: looks like your formatting is off again @sgrg1 ... there should be a space between the comma and month.

File: src/components/PaymentField.jsx [hunk 2]
```diff
@@ -40,7 +40,7 @@ const CustomInputComponent = ({
         <div className="tt_currency-picker">
           <CurrencyInput
             isNegativeValue={isNegativeValue}
-            id={name}
+            id={`${name}_${month}`}
```
Comment (martypowell) [inline src/components/PaymentField.jsx]: can be consistent and make this `${name}-${month}` please?

File: src/components/PaymentField.jsx (hunk 1 above)
Comment (martypowell) [reply src/components/PaymentField.jsx]: ok, we are actually already getting date, so we should use that, not sure what impact removing that will have. See below for more info.

File: src/components/PaymentField.jsx (hunk 2 above)
Comment (martypowell) [reply src/components/PaymentField.jsx]: you should be able to get month from the date prop. date.getMonth() should give you want you want. Then you don't have to change the props and the MonthlyPaymentFormStuff

File: src/components/PaymentField.jsx (hunk 1 above)
Comment (sgrg1) [reply src/components/PaymentField.jsx]: fixed

File: src/components/PaymentField.jsx (hunk 2 above)
Comment (sgrg1) [reply src/components/PaymentField.jsx]: fixed

File: src/components/PaymentField.jsx [hunk 3]
```diff
@@ -14,6 +14,7 @@ const CustomInputComponent = ({
 }) => {
   const { name } = field;
   const { className, isNegativeValue, label } = props;
+  const month = props.date.getMonth();
```
Comment (martypowell) [inline src/components/PaymentField.jsx]: you could have destructed on `date` on line 16, but not worth blocking.

**Pull request merged**